podvm/README.md

```diff
@@ -0,0 +1,134 @@
+# Introduction
+
+This directory contains the sources to build the podvm image (qcow2 file) for various Linux distributions and cloud providers. So use
+the instructions on the next sections if you need to build your own image with changes to meet your requirements. Otherwise you can
```

Comment:
nit: "So use the instructions on the next sections" => "So use the instructions **in** the next sections"